source/configuration_manual/authentication/user_database_extra_fields.rst

```diff
@@ -22,6 +22,10 @@ fields'. Possibilities are:
   for returning a variable number of fields (e.g. SQL).
 * ``uidgid_file``: Get uid and gid for user based on the given filename.
 
+.. versionchanged:: v3.0.0;v2.4.0
+   Extra fields can now also be set to empty string;
+   extra fields without value will default to empty string as well.
```

Comment:
Is this true? I thought the code still uses "yes" for these on the auth process side? Also it's not very clear what "without value" means, maybe "without '='"